<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>810</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>810</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>280</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>791</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="6">
     <widget class="QLabel" name="offsetYLabel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Offset Y:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="MultipleOfEightSpinBox" name="offsetXSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="widthLabelPx">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>px</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3">
     <widget class="MultipleOfEightSpinBox" name="heightSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLabel" name="offsetXLabel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Offset X:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="6" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="colorRadioButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Color</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">colorButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bwRadioButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>B/W</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">colorButtonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="8">
     <widget class="QLabel" name="offsetXLabelPx">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>px</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QLabel" name="offsetYLabelPx">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>px</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="3">
     <widget class="QCheckBox" name="lockCheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Lock to external trigger</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="camerasLabel">
      <property name="text">
       <string>Available cameras:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="heightLabelPx">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>px</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="widthLabel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Width:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="4">
     <widget class="QListWidget" name="camerasList">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="7">
     <widget class="MultipleOfEightSpinBox" name="offsetYSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="heightLabel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="MultipleOfEightSpinBox" name="widthSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="9">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="folderLabel">
        <property name="text">
         <string>Data storage path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MultipleOfEightSpinBox</class>
   <extends>QSpinBox</extends>
   <header>multipleofeightspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>camerasList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>onCamSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widthSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>onWidthChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offsetXSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>onOffsetXChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offsetYSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>onOffsetYChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>heightSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>onHeightChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lockCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>onLockToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>onColorToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>668</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>onBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>752</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCamSelectionChanged()</slot>
  <slot>onWidthChanged(int)</slot>
  <slot>onHeightChanged(int)</slot>
  <slot>onOffsetXChanged(int)</slot>
  <slot>onOffsetYChanged(int)</slot>
  <slot>onLockToggled(bool)</slot>
  <slot>onColorToggled(bool)</slot>
  <slot>onBrowse()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="colorButtonGroup"/>
 </buttongroups>
</ui>
